people/2015/spring/mellolikejello.yaml

```diff
@@ -22,6 +22,7 @@ hw:
   commarchpreso: https://mellolikejello.wordpress.com/2015/03/11/commarch-presentation/
   curriculum: https://mellolikejello.wordpress.com/2015/03/18/curriculum/
   teamprop2: https://mellolikejello.wordpress.com/2015/03/20/anglegators-team-proposal/
+  projteam: amm4108, RedTwo
```

Comment:
I think that makes more sense as a blog post. 



But in the case it is supposed to be a list I think you should do an actual list, maybe something like:

```
  projteam: ['amm4108', 'RedTwo']
```